lib/slash-commands/ext/ext.js

```diff
@@ -41,31 +52,96 @@ class Ext {
             callback = arguments[1];
         }
 
-        Util.readJsonFromFile(FILE, (err, data) => {
-            if (err) {
-                throw err;
+        if (this._attemptedToReadSpreadsheetConfigOnce) {//if we already tried to load the config file on a previous request...
+            this._handle(searchString, callback);//handle the request
```

Comment:
I can't do that because of the asynchronicity of the methods. Inside of your `if (never tried to load the config file)` condition i'm firing an asynchronous operation, so the line `this._handle(searchString, callback);` would probably execute before the `// config / json logic`.
But anyway, I'm changing this method a little bit for another reason.